docs/reference/search/request/scroll.asciidoc

```diff
@@ -165,23 +165,24 @@ DELETE /_search/scroll
 // CONSOLE
 // TEST[catch:missing]
 
-All search contexts can be cleared with the `_all` parameter:
+The recommended way to provide the `scroll_id` is by placing it in the request
+body using the `POST` verb against the `clear_scroll` endpoint added[5.2.0].
+All of the other ways to provide the `scroll_id`, either as a url parameter or
+as part of the request body using the `DELETE /_search/scroll` endpoint, are
+ deprecated deprecated[5.2.0].
 
-[source,js]
----------------------------------------
-DELETE /_search/scroll/_all
----------------------------------------
-// CONSOLE
-
-The `scroll_id` can also be passed as a query string parameter or in the request body.
-Multiple scroll IDs can be passed as comma separated values:
+All search contexts can be cleared with the special `_all` scroll id:
 
 [source,js]
 ---------------------------------------
-DELETE /_search/scroll/DXF1ZXJ5QW5kRmV0Y2gBAAAAAAAAAD4WYm9laVYtZndUQlNsdDcwakFMNjU1QQ==,DnF1ZXJ5VGhlbkZldGNoBQAAAAAAAAABFmtSWWRRWUJrU2o2ZExpSGJCVmQxYUEAAAAAAAAAAxZrUllkUVlCa1NqNmRMaUhiQlZkMWFBAAAAAAAAAAIWa1JZZFFZQmtTajZkTGlIYkJWZDFhQQAAAAAAAAAFFmtSWWRRWUJrU2o2ZExpSGJCVmQxYUEAAAAAAAAABBZrUllkUVlCa1NqNmRMaUhiQlZkMWFB
+POST /_search/clear_scroll
+{
+    "scroll_id" : [
+      "_all"
+    ]
```

Comment:
this one looks a bit odd, looked better providing _all as part of the url, but maybe that's ok